docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,104 @@
+This guide was developed using a `Nokia G-240W-B` router. It should work for other models in the `G` series.
+
+### Device information
+
+- Device name: `Nokia G-240W-B`
+- Vendor: `Nokia`
+- Hardware version: `3FE56756ADBA`
+- Software version: `3FE47969AGCA21`
+- Chipset: `BCM6838`
+
+> Note:
+There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is accessible from the `LAN` network via `SSH`.
+- The `Raspberry Pi` is using static IP address, `IPv4` and `IPv6` as well.
+- The `ISP` provides `IPv6` address, prefix and gateway to the router.
+
+For this example, we will be using the `192.168.1.100` and `fe80::192:168:1:100` addresses for the following steps. You will need to substitute your own addresses in the following steps.
+
+Notice that the `IPv4` and `IPv6` addresses suffixes matches the number `xxx.168.1.100` and `xxxx::192:168:1:100` respectively, this is intentional to recognize the addresses were manually assigned in the `Raspberry Pi`.
+
+Worth to mention that the `IPv6` address that starts with `fe80` is the `link-local` address and it is used in this guide, the other addresses that start with `2806` are the global addresses and they are assigned by the router.
+
+### Grab your IPv4 and IPv6 address from your Raspberry Pi
+
+SSH into your `Raspberry Pi`. A quick `ifconfig eth0` (substituting `eth0` for the applicable network interface) will get you the address information:
+
+```bash
+eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
+        inet 192.168.1.100  netmask 255.255.255.0  broadcast 192.168.1.255
+        inet6 2806:103e:1f:2ace:22a1:de1d:e1bf:30dc  prefixlen 64  scopeid 0x0<global>
+        inet6 2806:103e:1f:2ace:7d21:7337:7817:2fb2  prefixlen 128  scopeid 0x0<global>
+        inet6 fe80::e6f3:5490:577e:4fed  prefixlen 64  scopeid 0x20<link>
+        inet6 fe80::192:168:1:100  prefixlen 64  scopeid 0x20<link>
+        ether e4:5f:01:cd:8e:e1  txqueuelen 1000  (Ethernet)
+        RX packets 4342  bytes 704995 (688.4 KiB)
+        RX errors 0  dropped 0  overruns 0  frame 0
+        TX packets 3376  bytes 661707 (646.1 KiB)
+        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
+```
+
+e.g. `inet 192.168.1.100  netmask 255.255.255.0` and `inet6 fe80::192:168:1:100 prefixlen 64`
+
+### IPv4 configuration
+
+Go to **Network** --> **LAN**
+
+1. Set `IPv4 address` to `192.168.1.254`
```

Comment:
I think there is no need to change the IPv4 address of the router. It might be `192.168.1.1` already by default.

Reply:
It depends of the ISP configuration, in my case 192.168.1.254 is the default GW. I was told that other routers (provided by same ISP are using 192.168.1.1). 
I mentioned that user can set 192.168.1.1 or 192.168.1.254 depending in their default configuration.

Reply:
I configured the router to set static IP to the Raspi, the screenshot isn't including that section; please take a look to the pic attached to this message.

![IMG_1275](https://user-images.githubusercontent.com/6373960/197361022-3315eb37-53b6-466e-8871-6d891588078f.PNG)
* Im using wiregard so I can connect to home lan via LTE.

Reply:
This is a valid configuration. I think it's a bit cumbersome to set it static on the device and set a static DHCP assignment on the router, but your mileage may vary.  
However, for this public guide I would not recommend others to do so, because it's an additional step they need to perform. It's easier to separate static and dynamic IP pools.

Reply:
I updated the IP address to x.x.1.10 for IPV4, with this change the IP address matches the range used by the DHCP IPV4.

Reply:
Marked as "resolved", please let me know if this is OK to you. :)